tests/integration_tests/test_tracegraph_elbo.py

```diff
@@ -230,29 +224,30 @@ def setUp(self):
         # beta prior hyperparameter
         self.alpha0 = Variable(torch.Tensor([1.0]))
         self.beta0 = Variable(torch.Tensor([1.0]))  # beta prior hyperparameter
-        self.data = []
-        self.data.append(Variable(torch.Tensor([0.0])))
-        self.data.append(Variable(torch.Tensor([1.0])))
-        self.data.append(Variable(torch.Tensor([1.0])))
-        self.data.append(Variable(torch.Tensor([1.0])))
+        self.data = Variable(torch.Tensor([[0.0], [1.0], [1.0], [1.0]]))
         self.n_data = len(self.data)
-        data_sum = self.data[0] + self.data[1] + self.data[2] + self.data[3]
+        data_sum = self.data.sum(0)
         self.alpha_n = self.alpha0 + data_sum  # posterior alpha
-        self.beta_n = self.beta0 - data_sum + \
-            Variable(torch.Tensor([self.n_data]))
-        # posterior beta
+        self.beta_n = self.beta0 - data_sum + Variable(torch.Tensor([self.n_data]))  # posterior beta
         self.log_alpha_n = torch.log(self.alpha_n)
         self.log_beta_n = torch.log(self.beta_n)
 
+    @pytest.mark.skipif(not dist.beta.reparameterized, reason='not implemented')
+    def test_elbo_reparameterized(self):
+        self.do_elbo_test(True, 3000, 0.95, 0.0007)
```

Comment:
can't you do fewer steps, lower momentum, and higher lr here?

Reply:
I was having trouble getting tests to pass with reparameterized gradients. I was hoping to merge this big PR and then follow up by focusing on getting the tests to pass.